content/resources/lessons/acoustics-indices/presentation/index.md

```diff
@@ -37,6 +37,10 @@ Marina D. A. Scarpelli - PhD candidate at QUT
 ![Communicationnetwork](./communication-network.png)
 
 </div>
+
+>Notes:
+>The different components of the soundscape are usually described as:
+>*Biophony:* sounds emitted by animals; *Geophony:* sounds emitted by geographical components of the landscape (e.g.: wind, rain, strems, etc); *Technophony:* sounds emitted by humans and its machines.
```

Comment:
- add a space between the `> ` and the notes
- wrap your lines at 80 chars
- add a empty line after the notes block